portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,28 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#sidebar {
+    height: 100%;
```

Comment:
nit: Please match indent as set in earlier definitions. Here and below.

Reply:
Fixed indentations for style.css and went ahead and edited indentation inconsistencies in the html files.